js/binance.js

```diff
@@ -691,6 +691,14 @@ module.exports = class binance extends Exchange {
                     'CMFUTURE': 'delivery',
                     'MINING': 'mining',
                 },
+                'networks': {
+                    'ERC20': 'ETH',
+                    'TRC20': 'TRX',
+                    'BNB': 'BNB',
```

Comment:
I would change this line to be

```
{
     'BEP2': 'BNB',
     'BEP20': 'BSC',
     ...
}
```

to unify even more chains
<img width="486" alt="Screenshot 2021-09-21 at 09 43 28" src="https://user-images.githubusercontent.com/14319357/134212436-4b1a602f-c512-411b-a0b9-d42b81558382.png">

Reply:
Yeap I will change that's sounds good